docs/plugin_developers_guide/09_packaging.rst

```diff
@@ -0,0 +1,256 @@
+.. Documentation of package for a Plugin
+
+Packaging Documentation
+=======================
+
+Below are the prerequisite files which we need them in Plugin repository itself.
+
+Common files
+------------
+
+- Description
+   Summary of a plugin. Also make sure description of plugin must be in a single line as we do not support multiline yet.
```

Comment:
The file named *Description* should contain a brief description of the plugin and will be used as the description for the package that is created.